docs/pages/eas/workflows/syntax.mdx

```diff
@@ -476,9 +476,15 @@ jobs:
 
 You can customize the behavior of a workflow &mdash; commands to execute, control flow, environment variables, build profile, app version, and more &mdash; based on the workflow run's context.
 
-### `after` and `needs`
+Use the `${{ <expression> }}` syntax to access context properties and functions. For example: `${{ github.ref_name }}` or `${{ needs.build_ios.outputs.build_id }}`.
```

Comment:
```suggestion
Use the `${{ expression }}` syntax to access context properties and functions. For example: `${{ github.ref_name }}` or `${{ needs.build_ios.outputs.build_id }}`.
```

I'd suggest avoiding `<` and `>` before and after an expression, as this sometimes causes confusion whether `<...>` is a valid syntax or not. You have already added an inline highlight around the expression syntax, and that should be sufficient.